CONTRIBUTING.md

```diff
@@ -95,7 +95,7 @@ Benchmarking and profiling of pre-configured scripts is available as follow:
 
 ```shell
 bazel run -c opt --config=linux //benchmark:profile_time -- [name]
-bazel run -c opt --config=linux //benchmark:profile_memory -- [name] [-p]
+bazel run -c opt --config=linux //benchmark:profile_memory -- [name]
```

Comment:
@tanaysd - big mistake on my side, `profile_memory` uses `mprof`, not scalene (see `benchmark/profile_memory.sh`) 😬 please revert this change. LGTM from my side after that! 👍🏼

Reply:
done.